test/runtime/gbt/comm/bigBulkTran.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM != ugni
```

Comment:
I think is a valuable "stress" test regardless of config (I think we should run it with gasnet and even comm=none)